src/components/i18n.js

```diff
@@ -22,22 +19,38 @@ const getMyCampusLanguage = () => {
     else {
         return (chrome.runtime.getManifest()).default_locale
     }
+
 }
 
+let messages;
```

Comment:
why is this not inside of the i18n object?

Reply:
why not?

Reply:
why not not?

Reply:
Because its not used outside of i18n.

Reply:
k